adr/ADR-32.md

```diff
@@ -197,18 +197,40 @@ Pending messages and bytes count should be updated when:
 - A new pull request is published - add a value of `request.batch_size` to the pending messages count and
 the value of `request.max_bytes` to the pending byte count.
 - A new user message is processed - subtract 1 from pending messages count and subtract message size from penging byte count.
-The message size (in bytes) is defined as: `len(msg.data) + len(msg.subject) + len(msg.reply)`
-- A pull request termination error is received - subtract the value of `Nats-Pending-Messages` header
-from pending messages count and subtract the value of `Nats-Pending-Bytes` from pending bytes count.
-
-###### JetStream error handling
-
-There are 2 kinds of errors which can be received as a response to a pull request:
-
-- Pull request termination error is any error containing `Nats-Pending-Messages` and `Nats-Pending-Bytes` headers.
-These should not be treated as errors by the application, but rather used to calculate the pending messages/bytes count.
-- Every other pull request error should be treated as terminal error - it should be telegraphed to the user (in langie-specific way),
-the `Consume()` execution should be stopped and subscription should be drained.
+- A pull request termination status is received containing `Nats-Pending-Messages` and `Nats-Pending-Bytes` headers, subtract the value of `Nats-Pending-Messages` header from pending messages count and subtract the value of `Nats-Pending-Bytes` from pending bytes count.
+  - 408 Request Timeout
+  - 409 Message Size Exceeds MaxBytes 
```

Comment:
There can be many more 409...

Reply:
I just check all status for the headers to future proof the code